content/java/concepts/math-methods/terms/rint/rint.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: '.rint()'
+Description: 'Returns the closest double value depending on the value of the argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.rint()`** returns the closest double value depending on the value of the argument.
+
+## Syntax
+
+```pseudo
+Math.rint(num)
+```
+
+The return type of the `.rint()` method and data type of the `num` parameter is a double.
```

Comment:
This sentence is a little clumsy. You could rewrite it as something like: "The `.rint()` method takes a double as an argument and returns a double."

Reply:
@SSwiniarski, done.